src/modules/core/components/MembersList/MembersListItem.tsx

```diff
@@ -193,9 +193,9 @@ const MembersListItem = <U extends AnyUser = AnyUser>(props: Props<U>) => {
               <UserMention hasLink={false} username={username} />
             </span>
           )}
-          <span className={styles.address}>
-            <MaskedAddress address={walletAddress} />
-          </span>
+          <div className={styles.address}>
+            <CopyableAddress>{colonyAddress}</CopyableAddress>
+          </div>
```

Comment:
Are we totally sure this change is correct?

![Screenshot from 2021-02-05 15-08-05](https://user-images.githubusercontent.com/1193222/107037636-06c8b580-67c4-11eb-842d-a0548ffd720f.png)

I feel like the previous version looked better imho

Reply:
In vol.2: `wallet address should be possible to copy-paste`

I agree with you in that the previous version looked better tho